code/go/internal/validator/spec.go

```diff
@@ -145,15 +145,17 @@ func (s Spec) rules(pkgType string, rootSpec spectypes.ItemSpec) validationRules
 		{fn: semantic.ValidateVersionIntegrity},
 		{fn: semantic.ValidateChangelogLinks},
 		{fn: semantic.ValidatePrerelease},
-		{fn: semantic.ValidateMinimumKibanaVersion},
+		{fn: semantic.WarnOn(semantic.ValidateMinimumKibanaVersion), until: semver.MustParse("3.0.0")},
+		{fn: semantic.ValidateMinimumKibanaVersion, since: semver.MustParse("3.0.0")},
```

Comment:
Ah! So until 3.0.0 is the same behaviour, that's perfect!
I didn't remember that `until` parameter :+1: